dll/win32/advapi32/reg/hkcr.c

```diff
@@ -17,6 +17,17 @@ WINE_DEFAULT_DEBUG_CHANNEL(reg);
 
 static const UNICODE_STRING HKLM_ClassesPath = RTL_CONSTANT_STRING(L"\\Registry\\Machine\\Software\\Classes");
 
+static
+BOOL
+ValueExists(_In_ HKEY hNormalKey, _In_ PUNICODE_STRING Name)
```

Comment:
by "normal" key do you want to insist it is a "true" registry key, and not an hkcu one?

Reply:
Correct. Do you want an assert?

Reply:
An assert should be fine I guess (since it would just verify that this helper doesn't get used wrongly in the future)